cli/secret-render-template

```diff
@@ -1,5 +1,5 @@
-{{- with secret "6553ccb2b7da580d7f6e7260" "dev" "/" }}
+{{- with secret "ae06a663-2740-49c5-b998-bf4ceca2c120" "dev" "/" `{"recursive":true}` }}
```

Comment:
If we need to add other args before `{"recursive":true}` then would it be a issue?

Reply:
Answered as above